File: app/src/main/java/com/bresiu/codechallenge/repository/mappers/PhotoModelToEntityMapper.java
package com.bresiu.codechallenge.repository.mappers;

import com.bresiu.codechallenge.data.entity.Photo;

public class PhotoModelToEntityMapper
		implements Mapper.ObjectMapper<Photo, com.bresiu.codechallenge.model.Photo> {

	@Override public Photo map(com.bresiu.codechallenge.model.Photo modelPhoto) {
		return new Photo(modelPhoto.id, modelPhoto.albumId, modelPhoto.title, modelPhoto.url,
				modelPhoto.thumbnailUrl);
	}
}

File: app/src/main/java/com/bresiu/codechallenge/repository/mappers/PhotoModelToEntityMapper.kt
package com.bresiu.codechallenge.repository.mappers

import com.bresiu.codechallenge.data.entity.Photo

class PhotoModelToEntityMapper : Mapper.ObjectMapper<Photo, com.bresiu.codechallenge.model.Photo> {

  override fun map(mappingItem: com.bresiu.codechallenge.model.Photo): Photo {
    return Photo(mappingItem.id.toLong(), mappingItem.albumId.toLong(), mappingItem.title, mappingItem.url,
        mappingItem.thumbnailUrl)
  }
}